### ISSUE TITLE
Hide emails when sending the calendar invite

### ISSUE THREAD

**Issue by nicolls1** [calcom/cal.com#15839]
### Is your proposal related to a problem?

I do not want people scheduling bookings to be able to see my employee's email addresses. Our users should use other means of communication rather than directly contacting the person they are meeting with. Our employee's email addresses being on the booking leads to users directly reaching out via these emails.

### Describe the solution you'd like

Here is how calendly does it:
https://help.calendly.com/hc/en-us/articles/14076393762583-How-to-hide-your-personal-email-use-a-no-reply-email-address

I would like an extra setting on the event type to say use a no-reply email address.

### Describe alternatives you've considered

I do not believe there is currently a way to hide emails since cal.com account emails are used.

### Additional context

None, I could not find any open or closed issues related to this.

### Requirement/Document

None, see how calendly does it here: https://help.calendly.com/hc/en-us/articles/14076393762583-How-to-hide-your-personal-email-use-a-no-reply-email-address

---
##### House rules
- If this issue has a `🚨 needs approval` label, don't start coding yet. Wait until a core member approves feature request by removing this label, then you can start coding.
  - For clarity: Non-core member issues automatically get the `🚨 needs approval` label.
  - Your feature ideas are invaluable to us! However, they undergo review to ensure alignment with the product's direction.

**Comment by Mihir867** [comment]
Hi, I would like to work on this, Do I need it to be assigned or I can start working ?

**Comment by Mihir867** [comment]
Do you want it just to show name and a email address - noreply@cal.com to someone who is booking the event ?

**Comment by neehhaa06** [comment]
## Yes, I think there is few steps can take in to solve this issue! :)

> Allow users to choose whether or not to include the attendee list in their invites.

> Turn on the ability to issue invitations from an email address that does not reply.

> BCC Functionality: Include a function that allows email addresses to be hidden in invitations.

> Establish a service account in order to issue invitations on an employee's behalf.

> Custom Privacy Settings: Permit each kind of event to have its own set of privacy preferences.

**Comment by TheHenrick** [comment]
I would like to see this as well.

**Comment by mirabellemorah** [comment]
me too

**Comment by MoezGholami** [comment]
It seems the functionality is added in #21005 and the issue is safe to close.
You just need to turn on the "Hide organizer's email" option in your event's advanced settings. I just tested it and it works.

**Comment by herzkerl** [comment]
Be aware, this (#21005) only hides the organizer's email, not those of other people that were also added to the event—for example, via round-robin or manual plus addressing.

**Comment by Udit-takkar** [comment]
@anikdhabal is this fixed?

**Comment by anikdhabal** [comment]
Yes, currently we provide an option to hide organizer email address for any bookings. So Closing this issue. Let us know if you've any doubts or that feature doesn't meet your requirements 🙏
